lib/api/response/get_account_status_response_result.dart

```diff
@@ -55,9 +55,13 @@ class GetAccountStatusResponse extends GetAccountStatusResponseModel {
   ///
   /// For parameters information refer to [GetAccountStatusRequest].
   /// Throws an [BaseAPIException] if API response contains an error.
-  static Future<GetAccountStatusReceive> fetchAccountStatusRaw() async {
+  static Future<GetAccountStatusReceive> fetchAccountStatusRaw({
+    String? loginId,
```

Comment:
If it fits in one line
```suggestion
    String? loginId
```